### ISSUE TITLE
survival analysis

### ISSUE THREAD

**Issue by Hchang77** [zhangxiaoyu11/OmiEmbed#3]
Hello
My major field is medicine and I am new to python code.

I try to analyze the RNA-seq and survival data with OmiEmbed.
RNA-seq data (A.tsv) and survival data (survival.tsv) are located in 'data' folder, respectively.
I ran train_test.py, but the result was 'FileNotFoundError: [Errno 2] No such file or directory: './data/labels.tsv'

What arguments are needed for survival analysis using one RNA-seq data and one survival data?

Best regards,

**Comment by wmrenr** [comment]
There are labels in my file.

**Comment by Hchang77** [comment]
Thanks for your reply.

I understand that RNA-seq data (A.tsv), survival data (survival.tsv)  and labels (labels.tsv) are needed for survival analysis.

What information does 'labels.tsv' contain? 
Event of survival was already included in survival.tsv (Death = 1, alive = 0)

Regards,

**Comment by wmrenr** [comment]
No,the labels contain sample and samp[le_type.samples ,for exsample ,the first row is TCGA-WH-A86K-01A ,  0.


>

**Comment by wmrenr** [comment]
I am also a beginner of pytorch, can you leave a contact information, let's discuss it together？
